local-run/env.sh.sample

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Tomcat and pod provisioning configurations
+export IB_DOMAIN=".ngrok.io"
+export IB_HOST=my.static.subdomain.ngrok.io
+export IB_PORT=8080
+
+# Open Developer Platform endpoints
+# https://symphonyoss.atlassian.net/wiki/display/FM/Open+Developer+Platform
+export POD_HOST=foundation-dev.symphony.com
+export POD_PORT=443
+export AGENT_HOST=foundation-dev-api.symphony.com
+export AGENT_PORT=443
+export KM_HOST=foundation-dev.symphony.com
+export KM_PORT=443
+export KM_AUTH_HOST=foundation-dev-api.symphony.com
+export KM_AUTH_PORT=443
+export SM_HOST=foundation-dev-api.symphony.com
+export SM_PORT=443
+
+# Hubspot service account certificate
+export HUBSPOT_CERT_FILE=jira.p12
+export HUBSPOT_CERT_PWD=changeit
```

Comment:
The environment variables required by the application are:

export INTEGRATION_BRIDGE_DOMAIN=".ngrok.io"
export INTEGRATION_BRIDGE_HOST=myid.ngrok.io
export INTEGRATION_BRIDGE_HTTP_PORT=8080

export POD_HOST=foundation-dev.symphony.com
export POD_PORT=443
export AGENT_HOST=foundation-dev-api.symphony.com
export AGENT_PORT=443
export KEY_MANAGER_HOST=foundation-dev.symphony.com
export KEY_MANAGER_PORT=443
export KEY_MANAGER_AUTH_HOST=foundation-dev-api.symphony.com
export KEY_MANAGER_AUTH_PORT=443
export POD_SESSION_MANAGER_HOST=foundation-dev-api.symphony.com
export POD_SESSION_MANAGER_PORT=443

export APP_ID=hubspot
export CERT_FILE=hubspot.p12
export CERT_PWD=changeit